src/components/FindWallet/ResetFiltersButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from "next-i18next"
+import { BsArrowCounterclockwise } from "react-icons/bs"
+import { Center, Icon } from "@chakra-ui/react"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+
+type WalletEmptyStateProps = {
+  resetFilters: () => void
+  resetWalletFilter: React.MutableRefObject<() => void>
+}
+
+export const ResetFiltersButton = ({
+  resetFilters,
+  resetWalletFilter,
+}: WalletEmptyStateProps) => {
+  const { t } = useTranslation("page-wallets-find-wallet")
+
+  return (
+    <Center
+      as="button"
+      color="primary.base"
+      fontSize="xs"
+      gap={1}
+      _hover={{
+        color: "primary.hover",
+      }}
+      onClick={() => {
+        resetFilters()
+        resetWalletFilter.current()
+        trackCustomEvent({
+          eventCategory: "WalletFilterSidebar",
+          eventAction: `Reset button`,
+          eventName: `reset_click`,
+        })
+      }}
+    >
+      <Icon as={BsArrowCounterclockwise} aria-hidden="true" fontSize="sm" />
+      {t("page-find-wallet-reset-filters").toUpperCase()}
```

Comment:
Small non-blocking nit pick, to avoid JS for styling, might suggest textTransform here:

```suggestion
      textTransform="uppercase"
    >
      <Icon as={BsArrowCounterclockwise} aria-hidden="true" fontSize="sm" />
      {t("page-find-wallet-reset-filters")}
```